remoteconfig_test.go

```diff
@@ -10,7 +10,9 @@ import (
 	"strings"
 	"testing"
 
+	"github.com/golang/groupcache/lru"
```

Comment:
Can we try to rewrite this test to not use this dependency? It's not being used in any non-test code in this library, and so it seems less than ideal to make this a dependency that would then be pulled into every project that uses this library.